Pocket/pocket-ios
# fix(readableWidth) Fix readable width changing .compact views

## Summary
Toggles readableWidth based on size class. 

## References 
https://pocket.slack.com/archives/C4DL4QSU8/p1686253360513269?thread_ts=1685817699.582119&cid=C4DL4QSU8

## Implementation Details
Hold the Layout for Items and alter it's ContentInsetReference when the screen is being laid out.

## Test Steps
Similar to the Readable width changes, try on iPad and enter split screen.
Check on phone for uniformity with current app store release

## PR Checklist:

## Screenshots
![Simulator Screen Recording - iPad Air (5th generation) - 2023-06-15 at 14 37 08](https://github.com/Pocket/pocket-ios/assets/8590135/f574e99b-a7da-475a-be67-552131017fbe)
<img width="564" alt="Screenshot 2023-06-15 at 2 38 00 PM" src="https://github.com/Pocket/pocket-ios/assets/8590135/68dd53f5-ef4e-4f95-9285-d419bf46b33c">
